pkgs/development/libraries/libxslt/default.nix

```diff
@@ -41,7 +41,10 @@ stdenv.mkDerivation rec {
   propagatedBuildInputs = [ findXMLCatalogs ];
 
   # TODO move cryptoSupport as last flag, when upgrading libxslt
-  configureFlags = optional (!cryptoSupport) "--without-crypto" ++ [
+  configureFlags = []
+    ++ optional (buildPlatform != hostPlatform) "--with-libxml-prefix=${libxml2.dev}"
+    ++ optional (!cryptoSupport) "--without-crypto"
```

Comment:
We might as well put these on the end for the nice `= [` idiomatic line since we're mass rebuilding today anyways.